src/components/Branding/Branding.jsx

```diff
@@ -0,0 +1,15 @@
+import logo from 'assets/logo.svg'
+import Styled from './Branding.styles'
+
+const Branding = () => {
+  return (
+    <>
+      <Styled.Logo>
+        <img data-testid="logo" src={logo} className="logo" alt="logo" />
+      </Styled.Logo>
+      <Styled.Title data-testid="name">Ecommerce</Styled.Title>
```

Comment:
Podrías importar estilo por estilo, para evitar usar Styled.Componente, con esto el código se verá mas limpio.

Reply:
De acuerdo, justo lo hice de esta manera en objeto para evitar importar estilo por estilo, de igual forma los puedo renombrar en el siguiente entregable de la siguiente manera e importarlos individualmente
 
```suggestion
      import { TitleStyled, LogoStyled } from './Branding.styles'
      <TitleStyled data-testid="name">Ecommerce</TitleStyled>
```